arbitrum-docs/for-devs/concepts/public-chains.mdx

```diff
@@ -22,10 +22,15 @@ Arbitrum chains are Layer 2 solutions built on top of the Ethereum blockchain, d
 
 ## What Arbitrum testnet chains are available?
 
+### Arbitrum Sepolia
+
+Arbitrum Sepolia serves as a testnet chain replicating the capabilities of Arbitrum One's main network. Linked to the Sepolia testnet, it offers developers a secure platform to experiment with and evaluate their smart contracts prior to actual deployment on the mainnet.
+
 ### Arbitrum Goerli
 
 Arbitrum Goerli is a testnet chain that mirrors the functionality of the Arbitrum One mainnet. It's connected to the Ethereum Goerli testnet, allowing developers to test their smart contracts in a risk-free environment before deploying them on the mainnet.
 
+⚠️ Important note: Ethereum Goerli will be deprecated in Q1 2023 but will be supported until Q4 2023. Once Arbitrum Sepolia becomes publicly available, we will phase out the usage of Arbitrum Goerli. The support for Arbitrum Goerli will cease prior to the deprecation of L1 Goerli.
```

Comment:
Better to say Ethereum Goerli is depreciated on Q1 2023, but supports for it will be continued till Q4 2023

Reply:
changed! ty!